tests/stress/utils.py

```diff
@@ -12,8 +14,8 @@
 }
 
 
-def get_crm_resources(duthost, resource, status):
-    return duthost.get_crm_resources().get("main_resources").get(resource).get(status)
+def get_crm_resource_status(duthost, resource, status, namespace=DEFAULT_NAMESPACE):
```

Comment:
Hi @wenyiz2021, I did some search in the repo and found the following tests are using function `get_crm_resources()` (same function name from various places):
- arp/test_stress_arp.py
- crm/test_crm.py
- fdb/test_fdb_mac_move.py
- vxlan/test_vxlan_crm.py
- vxlan/test_vxlan_ecmp.py

Then I ran these tests and can confirm they are still working as expected.

Reply:
Hi, @cyw233 , should we change another test scripts to support T2 as well?

Reply:
Hey @yutongzhang-microsoft, we have checked all the skipped tests on T2 and documented the ones that indeed need to be supported on T2. For example, this `test_stress_routes` should run on T2, while `test_stress_arp` should always be skipped on T2 because it's T0 only. 
This is the very beginning of the T2 test gap and we will make more test cases supported on T2, for example, supporting `test_bfd.py` on T2 as my very next ticket.

Reply:
@cyw233 , I mean not only need to make sure the other tests are passing, but also need to change the caller places where were using `get_crm_resources`, change to use `get_crm_resource_status`. Or, another option is to not change the function name.
the tests you mentioned above were passing, they did not run on T2, but only run on T0 etc, `crm/test_crm.py` didn't call this function. So this PR will break these tests on T0